django_graph_api/graphql/schema.py

```diff
@@ -125,4 +127,27 @@ def execute(self, request):
             },
             variables=request.variables,
         )
+
+        errors = self.validate(request)
+        if errors:
+            return None, errors
+
         return query_root.execute()
+
+    def validate(self, request):
+        query_root = self.query_root_class(
```

Comment:
What would you think of using the query root instance created in `execute` instead of making a new one? Is that possible?